src/components/pages/get-started/architecture/architecture.jsx

```diff
@@ -3,21 +3,28 @@ import React from 'react';
 import Container from 'components/shared/container/container';
 import Heading from 'components/shared/heading';
 
-import diagram from './images/diagram.svg';
-
 const title = 'Architecture';
 const description =
-  'Cilium consists of an agent running on all cluster nodes and servers in your environment.  It provides networking, security, and observability to the workloads running on that node. Workloads can be containerized or running natively on the system.';
-
+  "Cilium comprises four key components: the Cilium agent, the Cilium client command line tool, the Cilium operator, and the Cilium CNI plugin. The agent, running on all cluster nodes, configures networking, load balancing, policies, and monitoring via Kubernetes or APIs that describe networking, service load-balancing, network policies, and visibility & monitoring requirements. The client tool, bundled with the agent, inspects and manages the local agent's status, offering direct access to eBPF maps. The operator centrally manages cluster tasks, handling them collectively rather than per node. The CNI plugin, invoked by Kubernetes during pod scheduling or termination, interacts with the node's Cilium API to configure necessary datapaths for networking, load balancing, and network policies.";
```

Comment:
did you pull this from somewhere?

Reply:
I wrote this. But it's basically a summary of the component overview section in the docs
https://docs.cilium.io/en/stable/overview/component-overview/